sphinx/environment/collectors/asset.py

```diff
@@ -64,9 +64,6 @@ def process_doc(self, app, doctree):
                 candidates['?'] = imguri
                 continue
             elif imguri.find('://') != -1:
-                logger.warning('nonlocal image URI found: %s' % imguri,
-                               location=node,
-                               type='image', subtype='nonlocal_uri')
```

Comment:
Now I dropped this warning because sphinx downloads all images using `ImageDownloader`.